internal/cmd/deploy/deploy.go

```diff
@@ -33,7 +42,23 @@ If you don't specify a stack name, rain will use the template filename minus its
 
 If a template needs to be packaged before it can be deployed, rain will package the template first.
 Rain will attempt to create an S3 bucket to store artifacts that it packages and deploys.
-The bucket's name will be of the format rain-artifacts-<AWS account id>-<AWS region>`,
+The bucket's name will be of the format rain-artifacts-<AWS account id>-<AWS region>.
+
+The CLI JSON file (--cli-json-file <string>) can be used to programmatically set tags and parameters.
```

Comment:
`--cli-json-file` seems a bit long-winded to me. I'd prefer we take even more inspiration from CodePipline and refer to this file as "template configuration" and use `-c` and `--config` for it.

Reply:
True, `-c` and `--config` is more concise!